test/unit/org/apache/cassandra/cql3/validation/entities/SecondaryIndexTest.java

```diff
@@ -1375,9 +1427,10 @@ public void testIndexOnFrozenUDT() throws Throwable
 
         execute("INSERT INTO %s (k, v) VALUES (?, ?)", 0, udt1);
         String indexName = createIndex("CREATE INDEX ON %s (v)");
+        assertTrue(waitForIndex(keyspace(), tableName, indexName));
```

Comment:
I think this is a feature and the whole point of the PR. Not getting writes until the index is ready. But having said that we don't have any commit log now that we changed to ignoring writes instead of failing them. So the only option left is what you say: we forward the writes to the index and he'll have to sort it out... Still it irks me and this behavior is not symmetrical  to the read path :shrug: